CodeNamesKeygen/app/backend/fio.php

```diff
@@ -0,0 +1,113 @@
+<?php
+define("CN_MAX_FILES",  20); //max files in directory (not used yet)
+define("CN_MAX_ACTIVE_GAMES", 10); //max concurrent games
+define("CN_MIN_TIME_TO_DELETE", 60*45);  //time after which an active game can be deleted
+    // Active game is a game that has the field state != 0x00000000
+    // (i.e. an empty board, but you can set some unused bit (except MSB) to indicate an active game)
+define("CN_MAX_TIME_TO_DELETE", 60*60*24); //time after a file will be deleted
+define("CN_FILE_PATH", "games");
+
+/**
+ * Return true if new game field can be created else return false
+ * 
+ * @return boolean true if new game can be created, else false
+ */
+function checkFolder(){
+    $files = scandir(CN_FILE_PATH);
+    $count_files = count($files) - 2;
+    $now = time();
+    $old_games = array();
+    $inactive_games = array(); //FIXME: redundant
+    $inactive_times = array();
+    $active_games = array();
+    for ($i = 2; $i<$count_files + 2; $i++){
+        $filename = CN_FILE_PATH."/".$files[$i];
+        $filetime = filemtime($filename);
+        $age = $now - $filetime;
+        if ($age > CN_MAX_TIME_TO_DELETE){
+            // older than 24 hours
+            $old_games[] = $filename;
+        }elseif($age > CN_MIN_TIME_TO_DELETE or !file_get_contents($filename)){
```

Comment:
Please add whitespaces everywhere in the code is presented in this suggestion

```suggestion
        } elseif ($age > CN_MIN_TIME_TO_DELETE or !file_get_contents($filename)) {
```

Also, extract this condition to function.

Reply:
Whitespace added. I see no point of the function as it would be used exactly once.

Reply:
Point is to describe what the condition is doing so nobody has to think about it.

Reply:
Would a comment suffice?

Reply:
Comments are a highway to hell, the whole point of the clean code is not have comments in the code.